configs/textdet/textsnake/textsnake_r50_fpn_unet_1200e_ctw1500.py

```diff
@@ -17,10 +17,7 @@
     neck=dict(
         type='FPN_UNet', in_channels=[256, 512, 1024, 2048], out_channels=32),
     bbox_head=dict(
-        type='TextSnakeHead',
-        in_channels=32,
-        text_repr_type='poly',
-        loss=dict(type='TextSnakeLoss')),
+        type='TextSnakeHead', in_channels=32, loss=dict(type='TextSnakeLoss')),
```

Comment:
miss postprocessor=dict(type=TextSnakePostprocessor', ****)

Reply:
It just uses the default one in `TextSnakeHead`. Mention it in the config file will be better?

Reply:
Yeah, configs should expose enough detail for users to quickly know the what parameters used and where to change